internal/operators/mtv/operator_test.go

```diff
@@ -0,0 +1,132 @@
+package mtv
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/operators/api"
+	"github.com/openshift/assisted-service/internal/operators/cnv"
+	"github.com/openshift/assisted-service/models"
+	"github.com/openshift/assisted-service/pkg/conversions"
+	"github.com/sirupsen/logrus"
+)
+
+var _ = Describe("MTV Operator", func() {
+	const (
+		openshiftVersion = "4.10.0"
+		minCpu           = 1
+		minRamMib        = 400
+	)
+
+	var (
+		log      = logrus.New()
+		operator api.Operator
+	)
+
+	Context("operator", func() {
+		BeforeEach(func() {
+			operator = NewMTVOperator(log)
+		})
+
+		It("should return the name of the operator", func() {
+			Expect(operator.GetName()).To(Equal(Name))
+			Expect(operator.GetFullName()).To(Equal(FullName))
+		})
+
+		It("should return the right validations ids", func() {
+			Expect(operator.GetClusterValidationID()).To(Equal(string(models.ClusterValidationIDMtvRequirementsSatisfied)))
+			Expect(operator.GetHostValidationID()).To(Equal(string(models.HostValidationIDMtvRequirementsSatisfied)))
+		})
+
+		It("should return the right feature support id", func() {
+			Expect(operator.GetFeatureSupportID()).To(Equal(models.FeatureSupportLevelIDMTV))
+		})
+	})
+
+	Context("host requirements", func() {
+		BeforeEach(func() {
+			operator = NewMTVOperator(log)
+		})
+
+		var cluster common.Cluster
+
+		BeforeEach(func() {
+			mode := models.ClusterHighAvailabilityModeFull
+			cluster = common.Cluster{
+				Cluster: models.Cluster{HighAvailabilityMode: &mode, OpenshiftVersion: openshiftVersion},
```

Comment:
```suggestion
				Cluster: models.Cluster{HighAvailabilityMode: &mode, OpenshiftVersion: nil},
```
remoeve openshift verison no needed